### PR TITLE
Rename steps and clean

### PR DESCRIPTION
**Pull request by ArthurNguyen**
step renaming in titles, filename, and variables in tasks

separate the file exists error from when overwrite is set to false + temporary print because I couldn't do a raise error correctly

### CODE REVIEW

File: eeg_flow/tasks/annotate_bad_channels_and_segments.py [hunk 1]
```diff
@@ -62,13 +62,15 @@ def bridges_and_autobads(
...
+        derivatives_folder / "plots" / f"{fname_stem}_step2_bridges.svg",
+        derivatives_folder / f"{fname_stem}_step2_with-bads_raw.fif",
     )
     locks = lock_files(*derivatives, timeout=timeout)
     try:
-        if not overwrite or all(derivative.exists() for derivative in derivatives):
+        if all(derivative.exists() for derivative in derivatives):
             raise FileExistsError
+        if not overwrite:
+            print("overwrite set to False")
```
Comment (mscheltienne) [inline eeg_flow/tasks/annotate_bad_channels_and_segments.py]: @ArthurNguyen Can you explain why you did this, so that I can clean-it up/improve correctly?

Comment (ArthurNguyen) [reply eeg_flow/tasks/annotate_bad_channels_and_segments.py]: if overwrite=False, it will raise the FileExistsError, even though the file may not exist

Comment (mscheltienne) [reply eeg_flow/tasks/annotate_bad_channels_and_segments.py]: oops yes, the condition is just wrong. (edit: should be and instead of or)

**Review by mscheltienne — APPROVED** [review]
I fixed some typos I've seen, and fixed the issue with overwrite.
Also since the bridges/autobads step is meant to be run in batch, I added a log of an exception instead of an hard interrupt.

A bit reluctant to merge this as is, could you run it once through and confirm that all is working and that we didn't miss a typo somewhere?

**Comment by ArthurNguyen** [comment]
will do thanks!

could we do a last edit: 

on step 3 annotations: in case file exists: write a new temp raw.fif (or warn that the file already exists)

the current behavior is that once the window is closed, any new annotations are lost. (because a file should not exist at this point)

Which makes sense, though is a pity in case someone started annotating a file that  already existed (for example, because it was created before, but not annotated) by mistake, then all is lost

**Comment by mscheltienne** [comment]
@ArthurNguyen Done

**Comment by ArthurNguyen** [comment]
merci!

**Comment by ArthurNguyen** [comment]
in the last commit: we avoid convreting the file again if it already exists, though the error message doesn't show up
(minor issue, I can survive without)

File: scripts/tasks/5_ICAs-select.ipynb [hunk 2]
```diff
@@ -61,10 +61,10 @@
...
    "source": [
-    "PARTICIPANT: str = \"\"  # e.g. \"P02\"\n",
-    "GROUP: str = \"\"  # e.g. \"G2\"\n",
-    "TASK: str = \"\"  # \"oddball\" or \"UT\"\n",
-    "RUN: int = 1  # 1 or 2"
+    "PARTICIPANT: str = \"P\"  # e.g. \"P02\"\n",
+    "GROUP: str       = \"G\"  # e.g. \"G2\"\n",
+    "TASK: str        = \"\"  # \"oddball\" or \"UT\"\n",
+    "RUN: int         = 1  # 1 or 2"
    ]
```
Comment (mscheltienne) [inline scripts/tasks/5_ICAs-select.ipynb:68]: @ArthurNguyen By the way, it's not black compliant ;)

**Pull request merged**